### ISSUE TITLE
More customisation options

### ISSUE THREAD

**Issue by eug48** [snapfeed/snapfeed-sdk#10]
IMHO it would be useful to be able to

- Disable the teaser
- Disable the draw feature
- Make Navigate the default option. I'm triggering feedback from a Feedback button in my Settings page so users would presumably need to navigate to the issue first up
- Add a "thank you" message

**Comment by flschweiger** [comment]
That are some great ideas! 

How do you feel about being able to specify a default behavior (e.g. showing a teaser or not, start with navigating vs drawing, etc.) inside the `SnapfeedConfig` and also being able to overwrite them in `Snapfeed.of(context).startFeedback(initialState: State.navigate, showTeaser: false)`?

**Comment by frasza** [comment]
I think that would be perfect!

**Comment by eug48** [comment]
Yes being able to override the config would be great. The app could have multiple buttons/icons to start feedback.

**Comment by frasza** [comment]
Additional idea:
Customisation of pen colours. Being able to define own colours for the Draw pen.
